src/styles/container.scss

```diff
@@ -0,0 +1,36 @@
+.container {
+  max-width: 944px;
+  padding: 50px 40px;
+  margin: 0 auto;
+
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  gap: 46px 48px;
+
+  justify-content: space-between;
+
+  align-items: center;
+
+  transition-duration: 0.3s;
```

Comment:
The transition-duration property is set to 0.3s, but it's unclear which properties this transition applies to. Ensure that transitions are applied to properties that benefit from smooth changes, such as 'transform' or 'opacity'.